src/system/utility.tsx

```diff
@@ -139,6 +146,63 @@ export const getRankId = (name: string) => {
   return 0;
 };
 
+
+
+
+
+
+
+
+// PlusMinus temporary hotfix
+//below
+export const arrayAnimalRanks = [
```

Comment:
We don't really need a lookup table for the rank images. I'll just store the ranks on the server using the ID instead.